task/configure-build/0.1/configure-build.yaml

```diff
@@ -33,6 +33,7 @@ spec:
           mountPath: /secret/default-push-secret
       script: |
         #!/usr/bin/env bash
+        set -e
```

Comment:
maybe `set -euo pipefail` to have all important failures covered?

Reply:
Looks unnecessarily for now in this script I think. `-e` is enough for catching potential problems as described in the JIRA issue. Option `pipefail` is added for some other script that has pipe operation, and `-u` is helpful along with the size of script is growing.

Reply:
okay